apps/vm/fsw/src/vm_app.cpp

```diff
@@ -519,6 +510,7 @@ int32 VM::RcvSchPipeMsg(int32 iBlocking)
                 SendVehicleManagerStateMsg();
                 SendVehicleControlModeMsg();
                 SendActuatorArmedMsg();
+                SendVehicleStatusMsg();
```

Comment:
Are the other additions of "SendVehicleStatusMsg()" necessary given it exists here?  Its better than its here so its in cyclic.

Reply:
This call should only exist here during wakeup now.